test/Test/Toml/Gen.hs

```diff
@@ -119,3 +129,33 @@ genToml = do
     kv     <- HashMap.fromList <$> genKeyAnyValueList
     tables <- Gen.list (Range.linear 0 10) genTableHeader
     pure $ TOML kv (fromList tables)
+
+genDay :: MonadGen m => m DateTime
```

Comment:
All these generators return `DateTime`. It's better if instead of they return types of `DateTime` field constructors and then you can write function:

```
genDate = Gen.choice [Day <$> getDay, Hours <$> genHours, ...]
```